src/compat-ccurl.c

```diff
@@ -21,7 +21,7 @@ char *ccurl_pow(char *trytes, int mwm)
         isInitialized = true;
     }
     pthread_mutex_unlock(&mtx);
-    return (char *) dcurl_entry((int8_t *) trytes, mwm);
+    return (char *) dcurl_entry((int8_t *) trytes, mwm, 0);
```

Comment:
I have no idea about the reason why it is defaulted to `0` instead of `1`?

Reply:
Oops, my bad.